modules/pixi/PixiLayerCustomData.js

```diff
@@ -227,7 +276,7 @@ export class PixiLayerCustomData extends AbstractLayer {
    * @param styleOverride Custom style
    */
   renderLines(frame, viewport, zoom, lines, styleOverride) {
-    const l10n = this.context.systems.l10n;
+    const l1n = this.context.systems.l1n;
```

Comment:
seems to be a typo in a bunch of places
```suggestion
    const l1n = this.context.systems.l10n;
```